server/src/api/v1/orders/get-orders.ts

```diff
@@ -65,6 +66,30 @@ export async function getOrders(req: Request, res: Response) {
     customResolver.set('is_active', (field: string, req: Request, value: string) => {
         return true;
     });
+    if (!Vars.currentUser.is_admin) {
+        let result: Customer[] = await Customer.findAll(
+            {
+                attributes: ['id'],
+                where: {
+                    userId: Vars.currentUser.id
+                },
+                raw: true
+            })
+            .catch(error => {
+                success = false;
+                return [];
+            });
+        if (!success) {
+            return res.status(500).send(wrapResponse(false, { error: 'Database error' }));
+        }
+        let customerIds: string[] = [];
+        result.forEach(el => {
+            customerIds.push(el.id);
+        });
+        customResolver.set('customerId', (field: string, req: Request, value: string) => {
```

Comment:
Getestet auch mit mehreren Customers? Bin mir gerade nicht schlüssig, ob das so klappt

Reply:
Ja, mein test user (nicht admin) hatte 4 customerIds, sodass das dann sowas gab in SQL:
SELECT "id", "customerId", "planId", "referrer", "consumption", "is_active", "terminatedAt", "createdAt", "updatedAt" FROM "Orders" AS "Order" WHERE "Order"."customerId" IN ('273f97d7-5400-4e9a-bf23-fdfb9098a255', '5f8dc784-ec0a-446f-a0d5-7c9b7f48841e', '55b50b2e-1cf6-48af-92c9-bf1d1beaa782', 'e1c65a81-2c94-42ae-95fe-a39932523896');
also genau wie benötigt

Reply:
Krass, wie gut das Sequelize auflöst (und mein QueryBuilder 😆)